crazyflie/src/crazyflie_server.cpp

```diff
@@ -160,34 +160,64 @@ class CrazyflieROS
     };
 
     if (enable_parameters) {
+      bool query_all_values_on_connect = node->get_parameter("firmware_params.query_all_values_on_connect").get_parameter_value().get<bool>();
+
       int numParams = 0;
       RCLCPP_INFO(logger_, "Requesting parameters...");
-      cf_.requestParamToc(/*forceNoCache*/);
+      cf_.requestParamToc(/*forceNoCache*/false, /*requestValues*/query_all_values_on_connect);
```

Comment:
Currently the only printout we get from this that we can determine whether the value retrieval is turned on or not is if the reqParamTOC takes longer than usual. Is it perhaps an idea to change or add the message that it's requesting parameters with or without values?

Reply:
yes indeed. I guess that should be enough feedback right?

Reply:
I think that is a very clear error message:-)